tests/acceptance/dcs.cf.sub

```diff
@@ -192,6 +193,9 @@ bundle agent default(filename)
 
       "bundles" slist => bundlesmatching("default:(init|test|check|destroy)");
 
+  files:
+      "$(G.testdir)/."    create => "true";
+
```

Comment:
I sense this might create trouble for tests that count number of files in the directory. But let's see what Jenkins says about it.